src/main/java/com/microsoft/sqlserver/jdbc/PerformanceActivity.java

```diff
@@ -0,0 +1,23 @@
+package com.microsoft.sqlserver.jdbc;
+
+public enum PerformanceActivity {

```

Comment:
I don't think we want to make these public.
```suggestion
enum PerformanceActivity {
```